input-data/offence-code/cjs-offences.json

```diff
@@ -961,7 +961,7 @@
   {
     "cjsCode": "AE71001",
     "offenceCategory": "CS",
-    "offenceTitle": "Employer fail to comply with an attachment of earnings order",
+    "offenceTitle": "Employer    fail to comply with an attachment of earnings order",
```

Comment:
I'd be keen to understand these white space changes

Reply:
This is wrong. Will investigate

Reply:
I've added consistent whitespace formatting to the CJS data too now. There is still a diff because it was wrong in main.